site-config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   themeColor: `#cc4141`,
   backgroundColor: `#fff`,
   pathPrefix: `/rules`,
-  logo: path.resolve(__dirname, `src/images/icon.png`),
+  // logo: path.resolve(__dirname, `src/images/icon.png`),
```

Comment:
delete this line

Reply:
logo was not being used anywhere so it was removed